hedera-node/hedera-app/src/test/java/com/hedera/node/app/blocks/impl/BlockImplUtilsTest.java

```diff
@@ -60,18 +61,20 @@ void testCombineWithNull() {
     @MethodSource("stateIdsByName")
     void stateIdsByNameAsExpected(@NonNull final String stateName, @NonNull final StateIdentifier stateId) {
         final var parts = stateName.split("\\.");
-        assertThat(BlockImplUtils.stateIdFor(parts[0], parts[1])).isEqualTo(stateId.protoOrdinal());
+        assertThat(StateUtils.stateIdFor(parts[0], parts[1])).isEqualTo(stateId.protoOrdinal());
     }
 
     public static Stream<Arguments> stateIdsByName() {
         return Arrays.stream(StateIdentifier.values())
-                .filter(v -> v != StateIdentifier.UNKNOWN)
+                .filter(v -> v != StateIdentifier.UNKNOWN && v != StateIdentifier.STATE_ID_VIRTUAL_MAP_STATE)
```

Comment:
@imalygin should changes related to the `STATE_ID_VIRTUAL_MAP_STATE` be only in `16785-D-virtualmap-keybytes-valueobjects`?